config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  get 'welcome/index'
+
+  get 'welcome/about'
```

Comment:
Consider making the about page available at the slightly friendlier & shorter URL path: `/about`